docs/content/concepts/visualizers-and-overrides.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Visualizer and Overrides
+order: 650
+---
+
+This section explains the process by which logged data is used to produce a visualization and how it can be customized via the user interface or code.
+
+*Note*: this area is under heavy development and subject to changes in future releases.
+
+## How are visualizations produced?
+
+[![image.png](https://i.postimg.cc/FRjRzX5g/image.png)](https://postimg.cc/47xZ2M3m)
+
+In the Rerun viewer, visualizations happen within _views_, which are defined by their [_blueprint_](blueprint.md).
+
+The first step for a view to display its content is to determine which entities are involved. This is determined 
+by the [entity query](../reference/entity-queries.md), which is part of the view blueprint. The query is run against the data store to generate the list of view entities.
+
+Views rely on visualizers to display each of their entities. For example, [3D views](../reference/types/views/spatial3d_view.md) use the `Points3D` visualizer to display 3D point clouds, and [time series views](../reference/types/views/time_series_view.md) use the `SeriesLine` visualizer to display time series line plots. Which visualizers are available is highly dependent on the specific kind of view. For example, the `SeriesLine` visualizer only exist for time series views—not, e.g., 3D views.
+
+For a given view, visualizers are selected for each of its entities based on their content. For example, in a 3D view, an entity containing a [`Position3D`](../reference/types/components/position3d.md) results in a `Points3D` visualizer being selected by default. (We will see that the visualizer selection process can be influenced by both the user interface and the blueprints.)
```

Comment:
This is slightly incorrect.

If an entity contains Position3D component it is compatible with the Points3D visualizer. To be compatible with Visualizer, an entity must have all of the required components.

The thing that makes it selected by default is the Points3DIndicator component that is also logged when you use the Points3D archetype.

If you just log Position3DBatch without the Points3D archetype, then you will have to instantiate the visualizer manually.

Reply:
Thanks. I wasn't sure if we had this strong indicator<->visualiser relationship in place already.